py/jpy-integration/build.gradle

```diff
@@ -82,54 +83,131 @@ idea {
   }
 }
 
-// only enable this if py building is enabled
-if (PyEnv.pythonEnabled(project)) {
+Task testJavaToPython = tasks.create 'testJavaToPython', {
+  Task t ->
+    t.group = 'python'
+    t.description = 'Run the java test suite that depends on deephaven-jpy (org.jpy.PyLib, etc)'
+    check.dependsOn t
+}
 
-  PyEnv env = PyEnv.getEnv(project)
+tasks.create 'debugJavaToPython', {
+  Task t ->
+    t.group = 'python'
+    t.description = 'Run the java exec io.deephaven.jpy.integration.PyDebug which is a sanity test for all JavaExec tasks'
+}
 
-  List<Task> pythonTests = []
-  Task testJavaToPython = tasks.create 'testJavaToPython', {
-    Task t ->
-      t.group = 'python'
-      t.description = 'Run the java test suite that depends on deephaven-jpy (org.jpy.PyLib, etc)'
-      check.dependsOn t
-      pythonTests += t
-  }
+tasks.create 'pathJavaToPython', {
+  Task t ->
+    t.group = 'python'
+    t.description = 'Run the java exec io.deephaven.jpy.integration.PySysPath which prints java properties and python paths'
+}
 
-  tasks.create 'debugJavaToPython', {
-    Task t ->
-      t.group = 'python'
-      t.description = 'Run the java exec io.deephaven.jpy.integration.PyDebug which is a sanity test for all JavaExec tasks'
-  }
+Task pythonToJava = tasks.create 'unittestPythonToJava', {
+  Task t ->
+    t.group = 'python'
+    t.description = 'Run "python -m xmlrunner discover" in src/python-to-java/python'
+    check.dependsOn t
+}
 
-  tasks.create 'pathJavaToPython', {
-    Task t ->
-      t.group = 'python'
-      t.description = 'Run the java exec io.deephaven.jpy.integration.PySysPath which prints java properties and python paths'
-  }
+if (!PyEnv.pythonEnabled(project)) {
+//  Task testTask = venvTest.javaTest(project, "java-to-python-test-${pv.name}", installWheels, sourceSets.javaToPython)
```

Comment:
"added" commented out code like this denotes that it hasnt been ported yet - this is a copy/paste from the pyEnabled section